Step02/.gitignore

```diff
@@ -0,0 +1,8 @@
+*.iml
+.gradle
+/local.properties
+/.idea/workspace.xml
+/.idea/libraries
+.DS_Store
+/build
+/captures
```

Comment:
Oh that was differ with the one that I used for assignment 1! Maybe.. I should have deleted both since we have one at the root directory which is effective on the subdirectories, too.